Services/GenerateHTML.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Aspose.Html;
+using FlightsCode.DBconfig;
+using FlightsCode.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FlightsCode.Services
+{
+    public class GenerateHTML
+    {
+        public void GenerateHTMLraport(List<Aircraft> printListAircrafts, string uploadDirectory)
```

Comment:
Truksta galimybiu isspausdinti ivairius raportus - tik EU, tik ne EU, visus. Praleista gera progra pasipraktikuoti skirtingus metodu overload'us. Be to, HTML kurimas ir raporto kurimas turetu buti atskirti - tai dvi skirtingos responsibilities. HTML kurimas turetu turetu savo servisa, raporto generavimas - savo.